src/NewProject/new_project_dialog.cpp

```diff
@@ -117,31 +157,38 @@ void newProjectDialog::Reset() {
   // Disable tab selection (by mouse and keyboard)
   ui->m_tabWidget->tabBar()->setAttribute(Qt::WA_TransparentForMouseEvents);
   ui->m_tabWidget->tabBar()->setFocusPolicy(Qt::NoFocus);
-
-  UpdateDialogView();
+  BackBtn->setVisible(true);
+  NextBtn->setVisible(true);
 }
 
-void newProjectDialog::UpdateDialogView() {
-  if (INDEX_LOCATION == m_index) {
-    BackBtn->setEnabled(false);
-  } else {
-    BackBtn->setEnabled(true);
-  }
+void newProjectDialog::ResetToProjectSettings() {
+  setWindowTitle(tr("Project settings"));
+  m_settings.clear();
 
-  if (INDEX_SUMMARYF == m_index) {
-    NextBtn->setEnabled(false);
-    ui->buttonBox->button(QDialogButtonBox::Ok)->setEnabled(true);
-    m_sumForm->setProjectName(m_locationForm->getProjectName(),
-                              m_proTypeForm->getProjectType());
-    m_sumForm->setDeviceInfo(m_devicePlanForm->getSelectedDevice());
-    m_sumForm->setSourceCount(m_addSrcForm->getFileData().count(),
-                              m_addConstrsForm->getFileData().count());
-  } else {
-    NextBtn->setEnabled(true);
-    ui->buttonBox->button(QDialogButtonBox::Ok)->setEnabled(false);
-  }
+  m_addSrcForm = new addSourceForm(this);
+  m_addSrcForm->SetTitle("Modify Design Files");
```

Comment:
```suggestion
  m_addSrcForm->SetTitle("Design Files");
```

Reply:
Remove the "Modify " from tab and titles. It implicit that users will modify the project using settings option.

Reply:
but this is literally modification. Are you sure?

Reply:
Yes, we don't need to explicitly mention it as opening up the settings implies that they are editable/modifiable.

Reply:
Done